.github/workflows/minikube.yaml

```diff
@@ -4,7 +4,7 @@ on:
     branches: [ main ]
 jobs:
   wait-for-images:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Solves the problems with minikube / oc due to https://www.github.com/actions/runner-images/issues/10636